### PR TITLE
Day6 exercises

### PR DESCRIPTION
**Pull request by dodoph**

### CODE REVIEW

File: Day6_excercise.rb [hunk 1]
```diff
@@ -0,0 +1,77 @@
+#1.store names of students
+names = ["Jessie", "Wendy", "Kelly"]
+#2.store students' names and their ages
+students = {Jessie: 29, Wendy: 22, Kelly: 33}
```
Comment (xsuoownit) [inline Day6_excercise.rb]: Do you need to add quotes to students' names?

File: Day6_excercise.rb [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+#3. store students' names, ages and their favorit colors.
+students = {
```
Comment (xsuoownit) [inline Day6_excercise.rb]: Don't know if the question asks, but do you need to be able to handle the case where there are duplicate names?

File: Day6_excercise.rb [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  Jessie: {age: 29, color: "white"},
+  Wendy: {age: 22, color: "Blue"},
+  Kelly: {age: 33, color: "green"}
+}
+# #4.store hashes with students' names, ages and their favorit colors in array
+students = [
+  {
+    name: "Jessie"
```
Comment (xsuoownit) [inline Day6_excercise.rb:14]: Do you need to add comma between fields?

Comment (dodoph) [reply Day6_excercise.rb:14]: don't need comma

File: Day6_excercise.rb (hunk 1 above)
Comment (dodoph) [reply Day6_excercise.rb]: Yes

File: Day6_excercise.rb (hunk 3 above)
Comment (xsuoownit) [reply Day6_excercise.rb:14]: 👍

File: Day6_excercise.rb (hunk 2 above)
Comment (dodoph) [reply Day6_excercise.rb]: it can store duplicate names

File: Day6_excercise.rb (hunk 1 above)
Comment (xsuoownit) [reply Day6_excercise.rb]: Okay, please fix it then.

**Review by xsuoownit — CHANGES_REQUESTED** [review]

File: Day6_excercise.rb (hunk 2 above)
Comment (xsuoownit) [reply Day6_excercise.rb]: Nope, with the hash structure, you can't store duplicate keys.

Comment (dodoph) [reply Day6_excercise.rb]: this is just an example that textbook ask to do, not a question

**Review by xsuoownit — APPROVED** [review]

**Comment by xsuoownit** [comment]
🚢

**Pull request merged**